app/model/editions/templates/TemplateHelpers.scala

```diff
@@ -94,30 +89,31 @@ trait EditionDefinition {
   val editionType: EditionType
   val notificationUTCOffset: Int
   val topic: String
-  val image: Option[SpecialEditionImage]
+  val buttonImageUri: Option[String]
   val expiry: Option[String]
   val buttonStyle: Option[SpecialEditionButtonStyles]
   val headerStyle: Option[SpecialEditionHeaderStyles]
+  val isLive: Boolean
```

Comment:
Is this needed?  Surely all Special and Regional Editions are live?  That's why we introduced Training: so that we could have test issues right through to the app.

Reply:
Yes I think it is needed at the moment- otherwise we need to fully support training editions in the app UI which would be a load of extra work. I know yesterday we agreed that rather than doing something like this we would just use Preview/Proof for testing, but at least at the moment I think there's a real use case (as in, we need it on the team right now) for having editions which exist in PROD and multiple issues available, but are hidden to users.

Reply:
Discussed in person - we decided `isLive` wasn't needed